README.md

```
@@ -90,15 +117,15 @@ Run `yarn` to install dependencies.
 
 ## Development server
 
-Run `yarn run nx serve {frontend | sr-frontend}` for a dev server. Navigate to http://localhost:4200/. The app will automatically reload if you change any of the source files.
+Run `yarn run nx serve {frontend | sr-frontend | zzz-frontend}` for a dev server. Navigate to <>http://localhost:4200/>. The app will automatically reload if you change any of the source files.
```

markdownlint-cli2 (0.17.2)

Bare URL used
null

(MD034, no-bare-urls)